wpilibc/athena/include/SpeedControllerGroup.h

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <functional>
+#include <vector>
+
+#include "SpeedController.h"
+
+namespace frc {
+
+class SpeedControllerGroup : public SpeedController {
+ public:
+  template <class... SpeedControllers>
+  explicit SpeedControllerGroup(SpeedController& speedController,
```

Comment:
Why is this templated, and taking a single reference then an elipsis? Also, we should have a version that takes an initializer list so we can initialize it like a normal array.